src/main/java/org/folio/rest/camunda/utility/OkapiRestTemplate.java

```diff
@@ -86,12 +90,14 @@ public ClientHttpResponse intercept(HttpRequest request, byte[] body, ClientHttp
           throws IOException {
         HttpHeaders headers = request.getHeaders();
 
-        headers.set(OKAPI_TOKEN_HEADER, token);
         headers.set(OKAPI_TENENT_HEADER, tenant);
+        headers.set(OKAPI_TOKEN_HEADER, token);
 
         headers.setAccept(Arrays.asList(APPLICATION_JSON, TEXT_PLAIN));
         headers.setContentType(APPLICATION_JSON);
 
+        LOG.info("Sending Okapi request: {} {}\nTenant: {}\nToken: {}", request.getMethod(), request.getURI(), tenant, token);
```

Comment:
I don't think we should log the token under INFO.
I think DEBUG would be acceptable, but the token should not appear in the logs if we can help it for security reasons.

Reply:
Right. Updating.